charts/argo-cd/templates/argocd-configs/argocd-cm.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.server.enabled }}
```

Comment:
I might be missing something, but I don't see any case where the server will ever be disabled. I think we should not have such a configuration at all.

Reply:
Resolved, see comment above.